projects/cm_mcu/Tasks.h

```diff
@@ -25,6 +25,12 @@
 #define __fp16 float
 #endif // __INTELLISENSE
 
+typedef union {
+  uint8_t us;
+  int8_t s;
+} convert_8_t;
+convert_8_t tmp1;
```

Comment:
you need to define `tmp1` in each file individually, and just put the typedef into the header